### DISCUSSION TITLE
Not auto selecting first completion from vscode

### DISCUSSION THREAD

**Discussion by tonyjara** [VSCodeVim/Vim#8293] [Q&A]
Hi, I really don't know how to re-create this or I'll post this as an issue. Randomly, sometimes I'm writing something and when I get a completion option, I press enter and it breaks the line. I have to manually do arrow down to get the completion. It's soooo annoyiing. Has anyone had this issue?

**Comment by dirtyhandscm** [comment]
Hey man I'm having the same issue! If i find a solution I'll you keep updated!

**Reply by tonyjara** [reply]
That'd be awesome, this is literally the only thing keeping it from being perfect for me
